sequelize/migrations/20250209210826-city-council-districts.js

```
@@ -0,0 +1,51 @@
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    try {
+      // eslint-disable-next-line max-len
+      await queryInterface.sequelize.query("ALTER TYPE enum_nyc_districts_type ADD VALUE 'city-council'");
+    } catch (e) {
+      // eslint-disable-next-line no-console
+      console.error(e);
```

Nitpick: In this case you'd probably want to return/throw, so you don't attempt to insert rows with an invalid value in the later parts of the function, right?

---

True. I removed the try-catch in a99eb6f7d231af7919d363c5badecc13f474a4d8 and instead checked for the existence of the value in the enum.